### ISSUE TITLE
Pass path to config.json as argument

### ISSUE THREAD

**Issue by laurentvd** [evanplaice/node-ftpsync#31]
Great work on the module. Really dig its simplicity. Small improvement request: being able to pass the filename of the config.json file. I have so many other stuff in the root of my project, and it would be great if we could change it. Other than that, love it!

**Comment by evanplaice** [comment]
@laurentvd Honestly, I was considering moving the config.json parameters into package.json. The only issue I see there is protecting the login credentials.

In the mean time it shouldn't be difficult to add the config filename as a configurable parameter.

**Comment by laurentvd** [comment]
Agreed. I think encouraging users to protect their credentials is a good thing. Maybe even suggest to put the default `config.json` in `.gitignore` and add a `config.example.json` to the VCS.
